frontend/src/pages/Search/Search.js

```diff
@@ -7,11 +7,13 @@ import 'pikaday/css/pikaday.css';
 import moment from 'moment';
 import './Search.css'
 import { Dropdown } from 'semantic-ui-react';
+import GeoSearch from '../../components/GeoSearch/GeoSearch';
 import { Link } from 'react-router-dom';
 import { SEARCH_RIDES_SUCCESS } from "../../actions/SearchPageStates";
 import { getRidesError, getRidesSuccess } from "../../reducers/SearchRidesReducer";
 
 const querystring = require('querystring');
+const DEBUG = true;
```

Comment:
will we be console.logging this in production mode?

Reply:
No, it's just for debugging. You can set it to false and it turns off all the console.log(...) lines, to prep for deployment